client/components/shipping/packages/packages-list-item.js

```diff
@@ -1,36 +1,56 @@
 import React, { PropTypes } from 'react';
 import Gridicon from 'components/gridicon';
 import Button from 'components/button';
+import classNames from 'classnames';
+
+const renderIcon = ( isLetter, isError ) => {
+	let icon;
+	if ( isError ) {
+		icon = 'notice';
+	} else {
+		icon = isLetter ? 'mail' : 'flip-horizontal';
+	}
+	return (
+		<Gridicon icon={ icon } size={ isError ? 29 : 18 } />
+	);
+}
 
 const PackagesListItem = ( {
 	index,
 	data,
 	dimensionUnit,
 	onRemove,
 	editPackage,
-} ) => (
-	<div className="wcc-shipping-packages-list-item">
-		<div className="package-type">
-			<Gridicon icon={ data.is_letter ? 'mail' : 'flip-horizontal' } size={ 18 } />
-		</div>
-		<div className="package-name">
-			<a href="#" onClick={ ( event ) => {
-				event.preventDefault();
-				editPackage( Object.assign( {}, data, { index } ) );
-			} }>
-				{ data.name }
-			</a>
-		</div>
-		<div className="package-dimensions">
-			<span>{ data.inner_dimensions } { dimensionUnit }</span>
-		</div>
-		<div className="package-actions">
-			<Button compact borderless className="remove-package" onClick={ onRemove }>
-				<Gridicon icon="cross-small" size={ 18 } />
-			</Button>
+	errors,
+} ) => {
+	const hasError = !! errors.length;
```

Comment:
`!!` is kinda sketchy as far as good practices go

also, why not just pass in `hasError` here directly since we never need the errors themselves in this component. I think the least info the components require, the better